easybuild/easyconfigs/m/MPICH/MPICH-3.3.2-GCC-9.3.0-2.34.eb

```
@@ -0,0 +1,14 @@
+name = 'MPICH'
+version = '3.3.2'
+
+homepage = 'http://www.mpich.org/'
```

@DinaMah Please use `https` here:

```suggestion
homepage = 'https://www.mpich.org/'